### PR TITLE
Require /unsafe flag for SkipLocalsInitAttribute

### PR DESCRIPTION
**Pull request by agocke**

### CODE REVIEW

File: src/Compilers/CSharp/Portable/Symbols/Attributes/AttributeData.cs [hunk 1]
```diff
@@ -664,6 +674,7 @@ internal bool ShouldEmitAttribute(Symbol target, bool isReturnType, bool emittin
 
             return true;
         }
+
```
Comment (RikkiGibson) [inline src/Compilers/CSharp/Portable/Symbols/Attributes/AttributeData.cs]: nit: blank line

File: src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs [hunk 2]
```diff
@@ -8929,6 +8929,49 @@ public static int Main()
...
+[SkipLocalsInitAttribute]
+public class C
+{
+    [SkipLocalsInitAttribute]
+    public void M()
+    { }
+
+    [SkipLocalsInitAttribute]
+    public int P => 0;
+}
```
Comment (RikkiGibson) [inline src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs]: Is this attribute supported on events?

Comment (jaredpar) [reply src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs]: In the event of that we should add a test to cover ... 

---
In reply to: [325397161](https://github.com/dotnet/roslyn/pull/38709#discussion_r325397161) [](ancestors = 325397161)

**Comment by jaredpar** [comment]
> }

Not blocking this PR but feel like we should factor out this repeated attributedefinition at some point. 

---
Refers to: src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs:10350 in 730f293. [](commit_id = 730f29347a33d18f92646f6b600663db07b94c11, deletion_comment = False)

**Comment by agocke** [comment]
I've confirmed that events are listed in the test plan, which should I confirm before merging. https://github.com/dotnet/roslyn/issues/25780

**Pull request merged**